### DISCUSSION TITLE
Receipt template

### DISCUSSION THREAD

**Discussion by jaahoo** [pay-rails/pay#732] [Q&A]
Hi, 
is there a way to style receipts? I have very specific design and I don't cut it with default pay receipt.
I might see two paths, first somehow extend Receipt gem with my own receipt model (is it possible to do that?) but I'm not sure if Prawn will do it.
Second way is to disable Receipts gem completely and use wicked_pdf to render html template receipts but how to integrate it with Pay gem?

**Comment by excid3** [comment]
Add a `receipt` method to the Pay::Charge model that renders a PDF and Pay will automatically include it as an attachment.

https://github.com/pay-rails/pay/blob/master/app/mailers/pay/user_mailer.rb

**Reply by jaahoo** [reply]
Wow that's elegant! Will try it.
Thank you Chris
